merge_coda_files/merge_coda_files.py

```diff
@@ -98,28 +132,30 @@ def coda_id_to_strings(list_td, old_key, key_map, code_scheme):
     code_scheme = open_scheme(coding_scheme_path)
 
     # Merge manually coded survey Coda files into the cleaned dataset
-    nr_label = CleaningUtils.make_cleaner_label(
-                code_scheme, 
-                code_scheme.get_code_with_control_code(Codes.NOT_REVIEWED),
-                Metadata.get_call_location()
-            )
+    #nr_label = CleaningUtils.make_cleaner_label(
```

Comment:
Think this commented code can now be removed

Reply:
Removed